twake/frontend/src/app/components/OnBoarding/AccountStatusComponent.tsx

```diff
@@ -35,9 +28,22 @@ const AccountStatusComponent = (): JSX.Element => {
       );
   };
 
+  const showUnverifiedModal = () => {
+    if (InitService.server_infos.auth?.console?.use === true)
+      return ModalManager.open(<UnverifiedAccount period={daysLeft} email={user.email} />, {
+        position: 'center',
+        size: { width: '600px' },
+      });
+  };
+
+  const showBanner = !user.is_verified;
+  if (!user.is_verified && daysLeft <= 0) {
```

Comment:
You just defined showBanner but you do not use it in this line

Reply:
updated